Anlab.Mvc/Client/components/PaymentSelection.tsx

```diff
@@ -56,27 +56,27 @@ export class PaymentSelection extends React.Component<IPaymentSelectionProps, IP
         return (
             <div>
 
-                <div className="flexrow">
-                    <div
-                        className={isCC ? activeDiv : inactiveDiv}
-                        onClick={() => this._handleChange("creditcard")}
-                    >
-                        <h3>Credit Card</h3>
-                        <p>It's amazing what you can do with a little love in your heart.</p>
-                    </div>
+                <div className="flexrow">
                     <div
                         className={isUcClient ? activeDiv : inactiveDiv}
                         onClick={() => this._handleChange("uc")}
                     >
                         <h3>UC Funds</h3>
-                        <p>It's amazing what you can do with a little love in your heart</p>
+                        <p>Required to receive the UC Rate.</p>
+                    </div>
+                    <div
+                        className={isCC ? activeDiv : inactiveDiv}
+                        onClick={() => this._handleChange("creditcard")}
+                    >
+                        <h3>Credit Card</h3>
+                        <p>Your credit card information will be collected at a later date.</p>
                     </div>
                     <div
                         className={isOther ? activeDiv : inactiveDiv}
                         onClick={() => this._handleChange("other")}
                     >
                         <h3>Other</h3>
-                        <p>It's amazing what you can do with a little love in your heart</p>
+                        <p>Payment by PO or Bank Transfer.</p>
```

Comment:
@jSylvestre I don't think we take bank transfers but if this is what they want it to say that's fine for now.